daemons/custom_query/constants.go

```diff
@@ -426,16 +426,18 @@ var StaticQueriesConfig = map[string]*QueryConfig{
 				EndpointType: "combined",
 				Handler:      "vyusd_price",
 				CombinedSources: map[string]string{
-					"ethereum":      "contract:ethereum",
-					"sushiswap_api": "rpc:sushiswapKatana",
-					"coingecko_api": "rpc:coingecko",
+					"ethereum":          "contract:ethereum",
+					"coingecko_api":     "rpc:coingecko",
+					"coinmarketcap_api": "rpc:coinmarketcap",
+					"coinbase_api":      "rpc:coinbase",
 				},
 				CombinedConfig: map[string]any{
-					"sushiswap_api_response_path": []string{"0x4772d2e014f9fc3a820c444e3313968e9a5c8121"},
-					"coingecko_api_response_path": []string{"yieldfi-ytoken", "usd"},
-					"coingecko_api_params": map[string]string{
-						"coin_id": "yieldfi-ytoken",
-					},
+					"coingecko_api_response_path":     []string{"usd-coin", "usd"},
```

Comment:
I think its better to use the priceCache that already supports usdc here, and you would only need to make a contract call here. Example: https://github.com/tellor-io/layer/blob/d0a00bb2469cd62cceda87e55ba8923660d7121b/daemons/custom_query/contracts/contract_handlers/yfield_yeth.go#L21-L61